bulk/repo-access-api.adoc

```diff
@@ -154,6 +164,14 @@ The repository SHOULD NOT validate the "meta-semantics" of the sent nodes, e.g.
 Rationale: We want to support storing invalid models, so the user does not lose data.
 Also, we assume no knowledge of the metamodel.
 
+[horizontal]
```

Comment:
My approach would be to separate model _storage_ (what we discuss here) from model _validation_. For storage, I'd aim at the least restrictions possible.